selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/taskmanagement/TaskManagementLocator.java

```diff
@@ -1,7 +1,14 @@
 package org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement;
```

Comment:
Just to be sure to understand, why this class existed before this PR ? Isn't this PR supposed to add task tests ?